app/javascript/react/components/Map/Map.tsx

```diff
@@ -59,24 +93,14 @@ const Map = () => {
     unit_symbol: unit_symbol,
   });
 
+  // Effects
```

Comment:
Does it add value to us? In one line below we can see the useCallback call

Reply:
Same as: https://github.com/HabitatMap/AirCasting/pull/660#discussion_r1633213995